css/syntaxes.json

```
@@ -57,7 +57,7 @@
     "syntax": "[ <line-names>? [ <fixed-size> | <fixed-repeat> ] ]* <line-names>? <auto-repeat>\n[ <line-names>? [ <fixed-size> | <fixed-repeat> ] ]* <line-names>?"
   },
   "axis": {
-    "syntax": "block | inline | vertical | horizontal"
+    "syntax": "block | inline | vertical | horizontal | x | y"
```

```suggestion
    "syntax": "block | inline | x | y"
```

see https://drafts.csswg.org/scroll-animations/#typedef-axis